python/pyrogue/_Root.py

```diff
@@ -63,7 +63,7 @@ def decrement(self):
 
     def _check(self):
         if len(self._list) != 0 and (self._count == 0 or (self._period != 0 and (time.time() - self._last) > self._period)):
-            #print(f"Update fired {time.time()}")
+            #print(f"Utpdate fired {time.time()}")
```

Comment:
@slacrherbst Need to fix this typo

Reply:
@slacrherbst Did you accidently delete `self._testFunc` in that commit while you were fixing the typo?

https://github.com/slaclab/rogue/pull/945/commits/e0176f3e94b1c96bc4ef9a7ea39aa39bfef2f45f